src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/AddressEntity.kt

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.digital.hmpps.personrecord.jpa.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.hibernate.envers.Audited
+import uk.gov.justice.digital.hmpps.personrecord.model.Person
+
+@Entity
+@Table(name = "address")
+@Audited
+class AddressEntity(
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
+
+  @Column(name = "address_line_one")
+  val addressLineOne: String? = null,
+
+  @Column(name = "address_line_two")
+  val addressLineTwo: String? = null,
+
+  @Column(name = "address_line_three")
+  val addressLineThree: String? = null,
+
+  @Column(name = "address_line_four")
+  val addressLineFour: String? = null,
+
+  @Column(name = "address_line_five")
+  val addressLineFive: String? = null,
+
```

Comment:
Is there anything to be said for storing address as just one string with newlines? It might facilitate comparisons with other systems. 5 lines of address is an arbitrary number anyway, what if one has six?

Reply:
we go with what we have currently and update in the future.